Core/Main/elxElastixFilter.h

```diff
@@ -188,9 +188,6 @@ class ELASTIXLIB_API ElastixFilter : public itk::ImageSource< TFixedImage >
   /** RemoveInputsOfType. */
   void RemoveInputsOfType( const DataObjectIdentifierType & inputName );
 
-  /** Let elastix handle input verification internally */
-  virtual void VerifyInputInformation( void ) ITK_OVERRIDE {};
-
```

Comment:
My fault  `:-)`  I declared the ITK5 `VerifyPreconditions()` member functions `const`: https://github.com/InsightSoftwareConsortium/ITK/commit/3e6b6f5bd6772316aa0fa6b89f31b42bef562112

Of course, if these functions are empty anyway, for `elastix`, it's fine to just remove them. 👍